.github/workflows/tox.yml

```diff
@@ -0,0 +1,18 @@
+name: tox
+on: [push, pull_request]
+jobs:
+  tox:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]  # [macos-latest, ubuntu-latest, windows-latest]
+        python: ['3.x']  # ['3.7', '3.8', '3.9', '3.10', '3.11-dev', 'pypy-3.9']
```

Comment:
We currently support 3.6-3.9 (and have issues with 3.10+). Should we limit python versions to only that subset?

Reply:
GitHub Actions ubuntu-latest no longer supports Python 3.6 because the Python Team no longer supports it.
* https://devguide.python.org/versions

I have set the tests to run on 3.7-3.9